cuppa/src/main/java/org/forgerock/cuppa/Runner.java

```diff
@@ -77,8 +78,8 @@ public Runner(Tags runTags) {
      * @param configuration Cuppa configuration to control the behaviour of the runner.
      */
     public Runner(Tags runTags, Configuration configuration) {
-        coreTestTransforms = Arrays.asList(new OnlyTestBlockFilter(), new TagTestBlockFilter(runTags),
-                new EmptyTestBlockFilter());
+        coreTestTransforms = Arrays.asList(new OnlyTestBlockFilter(), new ExpressionTagTestBlockFilter(runTags),
```

Comment:
I feel like there shouldn't be a need for 2 tag block filters. But saying that i'd like to look at trying to make changes to allow the groups expression to be an extension to Cuppa core. I will look at that separate to this PR though.